src/main/java/seedu/duke/apps/academicplanner/commands/EditModuleCommand.java

```diff
@@ -0,0 +1,62 @@
+package seedu.duke.apps.academicplanner.commands;
+
+import seedu.duke.apps.moduleloader.ModuleLoader;
+import seedu.duke.apps.academicplanner.commons.EditUtils;
+import seedu.duke.apps.academicplanner.commons.ModuleValidator;
+import seedu.duke.apps.academicplanner.exceptions.AcademicException;
+import seedu.duke.global.Command;
+import seedu.duke.global.objects.Person;
+import java.util.Scanner;
+
+//@@author harryleecp
+/**
+ * Class representing an edit module command from the academic planner.
+ */
+public class EditModuleCommand extends Command {
+    private static final String EDIT_SEMESTER = "1";
+    private static final String EDIT_GRADE = "2";
+    private static final String ERROR_INVALID_COMMAND = "INVALID COMMAND";
+    private static final String ERROR_NOT_ADDED = "You have not added this module into your list yet";
+    private static final String ERROR_EDIT_OPTION = "Number entered does not correspond to any feature";
+    public static final String PROMPT_EDIT_CHOICES = "Enter the number corresponding to the feature you wish to edit:\n"
+            + "\t1) Semester\n"
+            + "\t2) Grade";
+
+    private EditUtils editUtils;
+    private ModuleValidator moduleValidator;
+    private Scanner in;
+    private String moduleCode;
+
+    public EditModuleCommand(ModuleLoader allModules, Person currentPerson, Scanner in, String moduleCode) {
+        this.editUtils = new EditUtils(allModules, currentPerson);
+        this.moduleValidator = new ModuleValidator(allModules, currentPerson);
+        this.in = in;
+        this.moduleCode = moduleCode;
+    }
+
+    /**
+     * Allows user to edit the module entry of his academic calendar.
+     * Parameters allowed to change are semesterIndex or Grade.
+     */
+    @Override
+    public void execute() throws AcademicException {
+        try {
```

Comment:
Maybe it is alt instead of opt?
![image](https://user-images.githubusercontent.com/47108985/97398613-7a863700-1926-11eb-80a8-ffd4e0c840e4.png)